Declare yylex to fix:
y.tab.c:1585:16: error: implicit declaration of function 'yylex' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- cupl.y.orig	2015-11-21 17:59:23.000000000 -0600
+++ cupl.y	2023-01-12 19:27:59.000000000 -0600
@@ -34,6 +34,8 @@
 
 #include "cupl.h"
 
+int yylex(void);
+
 #ifdef YYBISON
 int yydebug;
 #endif /* YYBISON */
